just.config.js

```diff
@@ -47,10 +40,12 @@ task('install/holoflows', async () => {
  * @param cmd {string} The command you want to run
  * @param [opt] {object} Options
  * @param [opt.withWarn] {boolean} Show warn in stdio
+ * @param [opt.watch] {boolean} Watch continuously
  * @param [opt.env] {NodeJS.ProcessEnv} Environment key-value pairs
  */
-const step = (cmd, opt = { withWarn: process.env.CI === 'true' }) => {
-    const child = spawn(cmd, [], {
+const step = (cmd, opt = { withWarn: process.env.CI === 'true', watch: false }) => {
+    const spawn_cmd = opt.watch ? 'onchange "./src/**/*" -i -- ' + cmd : cmd
```

Comment:
As a general command, dose `watch` works for any command?

Reply:
`step` is a wrap for spawn, but `'onchange "./src/**/*" -i -- '` is only for prettier